.changes/next-release/enhancement-Endpoints-23278.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "enhancement",
+  "category": "Endpoints",
+  "description": "Allow usage of SSL common name with deprecation"
```

Comment:
For the description, let's update it to: "Enforce SSL common name as default for endpoint url". Mainly the wording is a bit more consistent with the change we are making as "Allow usage" is more indicative of adding an opt-in option or loosening a constraint.

Reply:
Got it! Will update.